testNestedForkJoinFailure
addNode
def

     f->(2,3,4)
     2->j
     3->j
     4->f2
     f2->(5,6)
     5-j2
     6-j2
     j-j2
     j2-end
    
j2
Expected to catch an exception but did not encounter any
f2
three
two
getCause
asList
we
invokeForkJoin
getErrorCode
assertTrue
four
dummyConf
end
five
six
f
one
getMessage
j
assertEquals
k
kill
[j2]
fail
contains
ex
parser
testWf
<worklfow-app/>
ErrorCode
Arrays
